lib/elixir/lib/regex.ex

```diff
@@ -348,6 +348,9 @@ defmodule Regex do
       order. Defaults to `:first` which means captures inside the regex do not
       affect the splitting process.
 
+    * `:include_captures` - when `true`, includes in the result the matches of
```

Comment:
Capture is very natural name in regex handling, also we already have :capture option in `run` function.
Maybe `with_` prefix since we use it for variable name?